_topic_map.yml

```diff
@@ -1168,6 +1168,8 @@ Topics:
     File: creating-a-multicomponent-application-with-odo
   - Name: Creating an application with a database
     File: creating-an-application-with-a-database
+  - Name: Running odo in a disconnected environment
```

Comment:
The rest of the OCP docs call this "restricted networks" not "disconnected environment". We should use the same terminology

Reply:
yep, thanks!